zeppelin-web/README.md

```diff
@@ -30,16 +30,29 @@ This will launch a Zeppelin WebApplication on port **9000** that will update on
 
 #### Troubleshooting
 
+**git error**
 In case of the error `ECMDERR Failed to execute "git ls-remote --tags --heads git://xxxxx", exit code of #128`
```

Comment:
Need a new line under those ** titles (git and proxy) otherwise the text is on the same line